# Individual waypoint with complete track information from CSV
float64 s_m          # Path length coordinate (frenet s)
float64 x_m          # X position in global frame
float64 y_m          # Y position in global frame  
float64 d_right      # Distance to right boundary
float64 d_left       # Distance to left boundary
float64 psi_rad      # Heading angle in radians
float64 kappa_radpm  # Curvature in rad/m
float64 vx_mps       # Velocity in m/s
float64 ax_mps2      # Acceleration in m/s^2
